community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -139,6 +139,10 @@
     public static final Setting<Long> transaction_start_timeout =
             setting( "transaction_start_timeout", DURATION, "1s" );
 
+    @Description("Location of the database plugin directory. Compiled Java JAR files that contain database " +
+                 "procedures will be loaded if they are placed in this directory.")
+    public static final Setting<File> plugin_dir = setting("dbms.plugin.directory", PATH, "plugins" );
```

Comment:
Because we don't want two separate folders for deploying extension code. Kernel extensions, Server Plugins and Unmanaged Extensions currently go in the `plugin` directory. It might make sense to rename that folder in 3.0, but I think that is something separate from this PR.